debian/wazo-nginx.dirs

```diff
@@ -2,3 +2,4 @@ etc/nginx/locations/http-available
 etc/nginx/locations/http-enabled
 etc/nginx/locations/https-available
 etc/nginx/locations/https-enabled
+var/www/html
```

Comment:
This is not necessary: the `install` file will take care of creating the directory

Reply:
ok